### PR TITLE
Exposing APIs into React Native

### PR DESCRIPTION
**Pull request by dcupidon**
I have implemented the annotationsListEditingEnabled, conversionOptions, navigationListForLargeDevices, restrictDownloadUsage and userBookmarksListEditingEnabled props.

### CODE REVIEW

File: src/DocumentView/DocumentView.js [hunk 1]
```diff
@@ -90,6 +90,11 @@ export default class DocumentView extends PureComponent {
     hideViewModeItems: PropTypes.array,
     pageStackEnabled: PropTypes.bool,
     showQuickNavigationButton: PropTypes.bool,
+    annotationsListEditingEnabled: PropTypes.bool,
+    conversionOptions: PropTypes.string,
+    navigationListForLargeDevices: PropTypes.bool,
+    restrictDownloadUsage: PropTypes.bool,
```
Comment (sgong-pdftron) [inline src/DocumentView/DocumentView.js:96]: @JamesTron same with this one, do you have suggestion for better naming?

**Review by sgong-pdftron — CHANGES_REQUESTED** [review]
- [ ] missing readme API

**Review by sgong-pdftron — APPROVED** [review]

File: API.md [hunk 2]
```diff
@@ -1080,6 +1080,17 @@ fields | array | array of field data in the format `{fieldName: string, fieldVal
 />
 ```
 
+#### annotationListEditingEnabled
+bool, optional, default value is true
+
+If the document editing is enabled, then this value determines whether the annotation list is readonly.
```
Comment (JamesTron) [inline API.md:1086]:
```suggestion
If document editing is enabled, then this value determines if the annotation list is editable.
```

File: API.md [hunk 3]
```diff
@@ -1101,6 +1112,17 @@ bookmarkJson | string | the list of current bookmarks in JSON format
 />
 ```
 
+#### userBookmarksListEditingEnabled
+bool, optional, defaults to true on Android and false on iOS
```
Comment (JamesTron) [inline API.md:1116]: > false on iOS

This doesn't sound right to me.

**Review by JamesTron — CHANGES_REQUESTED** [review]

File: src/DocumentView/DocumentView.js (hunk 1 above)
Comment (JamesTron) [reply src/DocumentView/DocumentView.js:95]: @sgong-pdftron I think we discussed all of these names, right? I think these are all OK.

**Review by JamesTron — APPROVED** [review]

**Pull request merged**